### DISCUSSION TITLE
商店でカーソルキー・テンキーでの上下移動での商品選択を可能にして欲しい。

### DISCUSSION THREAD

**Discussion by aigumagara** [hengband/hengband#3078] [Ideas]
そのままです。
個人的に商品とその値段が一致せず見にくいので選択した列が光ってくれると嬉しいです。

As it is.
Personally, I would be happy if the selected rows light up, as it is hard to see the products and the price do not match!

**Comment by deskull-m** [comment]
3年近く越しに失礼。Issueに上げておきました。クローズかけさせていただきます。
